src/js/hooks/cycleCount/useCountStep.jsx

```diff
@@ -31,20 +35,34 @@ const useCountStep = () => {
     cycleCountIds: state.cycleCount.requests,
     currentLocation: state.session.currentLocation,
   }));
-
-  useEffect(() => {
-    (async () => {
+  const fetchCycleCounts = async () => {
+    try {
+      show();
       const { data } = await cycleCountApi.getCycleCounts(
         currentLocation?.id,
         cycleCountIds,
       );
       tableData.current = data?.data;
+      // Date counted and assignee are the same for all items,
+      // so we create a map looking at first item
       const countedDates = data?.data?.reduce((acc, cycleCount) => ({
         ...acc,
         [cycleCount?.id]: cycleCount?.cycleCountItems[0].dateCounted,
       }), {});
       setDateCounted(countedDates);
-    })();
+      const countedByMap = data?.data?.reduce((acc, cycleCount) => ({
+        ...acc,
+        [cycleCount?.id]: cycleCount?.cycleCountItems[0].assignee,
+      }), {});
```

Comment:
btw, you could get the `countedDate` and `countedBy` in on reduce

Reply:
I did it initially, but ran into some problems with accessibility in the further state. I don't remember the exact context though. If you find time, you could try to do it.